STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/web/repository/upload/zip/DefaultZipEntryCommand.java

```diff
@@ -13,7 +14,7 @@ public boolean execute(String directoryPath) throws IOException {
     }
 
     @Override
-    public boolean execute(String filePath, InputStream inputStream) throws IOException {
+    public boolean execute(String filePath, InputStream inputStream) throws IOException, JAXBException {

```

Comment:
No needs to declare Exceptions in the interface implementation, if they are not throw there. Check it in all classes which was modified by you.